### PR TITLE
Use training data from a managed datasource

### PR DESCRIPTION
**Pull request by eedorenko**

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -122,6 +122,9 @@ Check out the newly created resources in the [Azure Portal](portal.azure.com):
 
 (Optional) To remove the resources created for this project you can use the [/environment_setup/iac-remove-environment.yml](../environment_setup/iac-remove-environment.yml) definition or you can just delete the resource group in the [Azure Portal](portal.azure.com).
 
+**Note:** The training ML pipeline uses a [sample diabetes dataset](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_diabetes.html) as training data. If you want to use your own dataset, you need to [create and resgister a datastore](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-access-data#azure-machine-learning-studio) in your ML workspace and put the datafile in the corresponding blob container (e.g. [diabetes.csv](./data/diabetes.csv)). Having done that, you'll need to configure DATASTORE_NAME and DATAFILE_NAME variables in ***devopsforai-aml-vg*** variable group.
```
Comment (dtzar) [inline docs/getting_started.md]: Let's also provide a link to the **az** cli option [here](https://docs.microsoft.com/en-us/cli/azure/ext/azure-cli-ml/ml/dataset?view=azure-cli-latest#ext-azure-cli-ml-az-ml-dataset-register).

**Comment by eedorenko** [comment]
> Have you tested this using a registered diabetes dataset in blob storage? We should use it as default for the CI pipeline - I don't see it setup there...

Yes, I have. With an ML WS in our subscription (EuUniqueML-AML-WS). I will setup that with the Eng subscription

File: docs/getting_started.md [hunk 2]
```diff
@@ -122,6 +122,10 @@ Check out the newly created resources in the [Azure Portal](portal.azure.com):
 
 (Optional) To remove the resources created for this project you can use the [/environment_setup/iac-remove-environment.yml](../environment_setup/iac-remove-environment.yml) definition or you can just delete the resource group in the [Azure Portal](portal.azure.com).
 
+**Note:** The training ML pipeline uses a [sample diabetes dataset](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_diabetes.html) as training data. If you want to use your own dataset, you need to [create and resgister a datastore](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-access-data#azure-machine-learning-studio) in your ML workspace and put the datafile in the corresponding blob container (e.g. [diabetes.csv](./data/diabetes.csv)). You can also define a datastore in the ML Workspace with [az cli](https://docs.microsoft.com/en-us/cli/azure/ext/azure-cli-ml/ml/datastore?view=azure-cli-latest#ext-azure-cli-ml-az-ml-datastore-attach-blob).
```
Comment (jotaylo) [inline docs/getting_started.md]: >resgister [](start = 234, length = 9)

nit: register

File: docs/getting_started.md [hunk 3]
```diff
@@ -122,6 +122,10 @@ Check out the newly created resources in the [Azure Portal](portal.azure.com):
 
 (Optional) To remove the resources created for this project you can use the [/environment_setup/iac-remove-environment.yml](../environment_setup/iac-remove-environment.yml) definition or you can just delete the resource group in the [Azure Portal](portal.azure.com).
 
+**Note:** The training ML pipeline uses a [sample diabetes dataset](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_diabetes.html) as training data. If you want to use your own dataset, you need to [create and resgister a datastore](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-access-data#azure-machine-learning-studio) in your ML workspace and put the datafile in the corresponding blob container (e.g. [diabetes.csv](./data/diabetes.csv)). You can also define a datastore in the ML Workspace with [az cli](https://docs.microsoft.com/en-us/cli/azure/ext/azure-cli-ml/ml/datastore?view=azure-cli-latest#ext-azure-cli-ml-az-ml-datastore-attach-blob). 
+Having done that, you'll need to configure DATASTORE_NAME and DATAFILE_NAME variables in ***devopsforai-aml-vg*** variable group.
```
Comment (jotaylo) [inline docs/getting_started.md]: >Having done that, you'll need [](start = 0, length = 29)

Consider changing this wording to "You'll also need to"

File: docs/getting_started.md (hunk 2 above)
Comment (jotaylo) [inline docs/getting_started.md]: >put the datafile in [](start = 388, length = 19)

Consider changing the wording to "upload the datafile (e.g. [diabetes.csv](./data/diabetes.csv)) to the corresponding blob container."

**Pull request merged**